sky/provision/kubernetes/instance.py

```diff
@@ -149,11 +195,13 @@ def _raise_pod_scheduling_errors(namespace, new_nodes):
                                  in event_message) or
                                 ('didn\'t match Pod\'s node affinity/selector'
                                  in event_message)):
-                                msg = lack_resource_msg.format(resource='GPU')
-                                raise config_lib.KubernetesError(
-                                    f'{msg} Verify if '
+                                extra_msg = (
+                                    f'Verify if '
                                     f'{pod.spec.node_selector[label_key]}'
                                     ' is available in the cluster.')
+                                raise config_lib.KubernetesError(
+                                    lack_resource_msg('GPU', pod, extra_msg,
+                                                      event_message))
```

Comment:
```suggestion
                                                     details=event_message))
```
for consistency with the above uses of `lack_resource_msg`